examples/production/values.yaml

```
@@ -2,6 +2,8 @@ global:
   telemetry:
     prometheus:
       enabled: true
+  spire:
+   productionChecks: false
```

```suggestion
   strictMode: true
```

As suggested earlier, maybe call it strictMode. Shouldn't the value be true for this test?

---

Good catch. broke in the last rename I guess.